payment-instruments/mobile-pay/seamless-view.md

```diff
@@ -0,0 +1,305 @@
+</html>
+```
+
+Lastly, initiate the Seamless View with a JavaScript call to open the `iframe`
+embedded on your website.
+
+{:.code-view-header}
+**JavaScript**
+
+```js
+<script language="javascript">
+    payex.hostedView.mobilepay({
+        // The container specifies which id the script will look for to host the
+        // iframe component.
+        container: "swedbank-pay-seamless-view-page"
+    }).open();
+</script>
+```
+
+## MobilePay Online Seamless flow
+
+The sequence diagram below shows the two requests you have to send to
+Swedbank Pay to make a purchase.
+The links will take you directly to the API description for the specific
+request.
+
+```mermaid
+sequenceDiagram
+    activate Merchant
+    Merchant->>-SwedbankPay: POST /psp/mobilepay/payments
+    activate SwedbankPay
+    note left of Merchant: First API request
+    SwedbankPay-->>-Merchant: Payment response with rel: view-payment
+    activate Merchant
+    Merchant->>-SwedbankPay: Script init of iFrame
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: Display Payment Page
+    activate Merchant
+    Merchant->>Merchant: Init payment
+    Merchant->>-SwedbankPay: Init request
+
+    activate SwedbankPay
+    SwedbankPay->>+MobilePay_API: POST <rel:create-authorization>
+    activate MobilePay_API
+    MobilePay_API-->>+SwedbankPay: Response
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: Display instructions page
+
+    MobilePay_API-->>-MobilePay_App: Confirm Payment UI
+    activate MobilePay_App
+    MobilePay_App-->>MobilePay_App: Confirmation Dialogue
+    MobilePay_App-->>-MobilePay_App: Confirmation
+
+    activate MobilePay_API
+    MobilePay_API->>-SwedbankPay: Make Payment
+    activate SwedbankPay
+    SwedbankPay-->>-SwedbankPay: Process Payment
+    activate SwedbankPay
+    SwedbankPay-->>-MobilePay_API: Process Payment Response
+    activate MobilePay_API
+    MobilePay_API-->>-MobilePay_App: Transaction Status
+
+    activate Merchant
+    Merchant->>- SwedbankPay: GET <MobilePay payment>
+    activate  SwedbankPay
+    SwedbankPay-->>-Merchant: Payment response
+    activate Merchant
+    Merchant-->>-Merchant: Display payment Status
+```
+
+1.  When the payer starts the purchase process, you make a `POST` request
+    towards Swedbank Pay with the collected Purchase information.
+2.  `rel: view-payment` is a value in one of the operations, sent as a response
+    from Swedbank Pay to the Merchant.
+3.  `Open iFrame` creates the Swedbank Pay Seamless View.
+    The Seamless View displays the payment page as content inside of the
+    `iFrame`.
+4.  A `POST` request is sent to the MobilePay API with
+    the mobile number (optional) for authorization.
+5.  Swedbank Pay handles the dialogue with MobilePay and the payer confirms the
+    purchase in the MobilePay app.
+6.  After the purchase has been authorized, the payer will be redirected back
+    to the merchant, where the status can be checked to ensure the payment was
+    successful.
```

Comment:
Note: Unsure what the numbers refer to, as I am unable to see any numbers in the Vipps diagram for reference, through the steps are more or less the same. I added `(optional)` to step 4, as without the MSISDN, the only change is that the user will have to put in the phone number themselves.

Reply:
The numbers are supposed to refer to steps in the diagram. Didn't we use to have ①②③ in the diagram before, @arebra?

Reply:
I based this off the Vipps diagram, which does not seem to have the numbering anymore.